bin/kong

```diff
@@ -1,16 +1,12 @@
-#!/usr/bin/env resty
+#!/usr/bin/env resty -c 65535
```

Comment:
This would override whatever `ulimit` I have already set - do we really need it?